aviary/modules/binning/binning.smk

```diff
@@ -749,7 +749,7 @@ rule checkm_das_tool:
 rule singlem_pipe_reads:
     output:
         "data/singlem_out/metagenome.combined_otu_table.csv"
-    threads: 1
+    threads: min(config["max_threads"], 48)
```

Comment:
Why do these rules keep the `min`?

Reply:
Because previously it was using pplacer_threads, or at least a variable called pplacer_threads. So I just figured if singlem does use pplacer then we should limit it with pplacer rules